src/routes.js

```diff
@@ -1417,6 +1420,101 @@ module.exports = async (
     sentPayment.write(paymentRequest);
   });
 
+  app.post("/api/lnd/trackpayment", async (req, res) => {
```

Comment:
client posts hash

Reply:
It's a POST because the client sends the payment's hash, we could include the payment hash as a param in the route's URL but I'm trying to maintain the same method values from lncli so it's easier to predict the URL by its equivalent lncli method name

Reply:
```
It's a POST because the client sends the payment's hash, we could include the payment hash as a param in the route's URL but I'm trying to maintain the same method values from lncli so it's easier to predict the URL by its equivalent lncli method name
```

t's cool to have route names match lnd method names, 

But, if this is for tracking an existing payment, this is obviusly a `GET`, we'll agree to disagree then.